cli/tests/integration_tests_lsp.rs

```diff
@@ -31,7 +33,10 @@ fn init(init_path: &str) -> LspClient {
   client
 }
 
-fn did_open<V>(client: &mut LspClient, params: V)
+fn did_open<V>(
+  client: &mut LspClient,
+  params: V,
+) -> Vec<PublishDiagnosticsParams>
```

Comment:
```suggestion
) -> Vec<lsp::PublishDiagnosticsParams>
```

And remove the use clause for it above.